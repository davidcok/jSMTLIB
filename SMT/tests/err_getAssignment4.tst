; get-assignment after unsat
(set-option :produce-assignments true)
(set-logic QF_UF)
(assert false)
(check-sat)
(get-assignment)
