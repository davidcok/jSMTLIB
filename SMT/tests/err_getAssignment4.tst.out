unsupported
success
success
unsat
(get-assignment)
^^^^^^^^^^^^^^^^
(error "The get-assignment command is only valid if :produce-assignments has been enabled")
